docs/sijopankki.md

```diff
@@ -2,7 +2,7 @@
 sort: 3
 ---
 
-# Uuden HELMET 4.0 sij19 -sijoittelupankin perustaminen ”tyhjästä”
+# Uuden HELMET 4.1 sij19 -sijoittelupankin perustaminen ”tyhjästä”
```

Comment:
Dimensions are a bit different from the table in this manual:
![kuva](https://user-images.githubusercontent.com/20718745/143898428-6a8ced48-3d30-47a8-bf7b-105933b30edf.png)